Первый таск

frozzzen123 opened the pull request:
Создал каркас для игрового интерфейса, добавил в проект файл со шрифтом, реализовал вывод на экран имен игроков.

frozzzen123 committed:
Update pythonReady.cpp

Создал каркас сцены и вывел имена игроков

frozzzen123 committed:
Add files via upload

fonts/Consolas.ttf

23350406 commented on code/pythonReady.cpp:
```diff
@@ -7,14 +7,102 @@
+			window.draw(_playersNames[i]);
+		}
+	}
+private:
```
кто пишет private внизу, непонятно вообще

23350406 commented on code/pythonReady.cpp:
```diff
@@ -7,14 +7,102 @@
+		_round = 0;
+	}
+	// Конструктор игрового интерфейса для многораундовой игры.
+	Layout(sf::Color color, unsigned numberOfPlayers, sf::Text* playersNames, unsigned round, unsigned* scores)
```
Не делать два конструктора

23350406 commented on code/pythonReady.cpp:
```diff
@@ -7,14 +7,102 @@
+		window.draw(line, 2, sf::Lines);
+
+
+		sf::Font consolas;
```
коммент на шрифты

23350406 commented on code/pythonReady.cpp:
```diff
@@ -26,6 +114,15 @@ int main()
 	pythonSprite.setTexture(pythonTexture);
 	pythonSprite.setPosition(420, 25);
 
+	sf::Font consolas;
```
комменты сюда плз

23350406 reviewed (commented):
Всё круто, но побольше комментов <3

frozzzen123 committed:
Исправление недочетов

Добавлены комментарии к блокам кода.
Конструкторы игрового интерфейса переписаны в один.
Реализованы Set и Get методы для игрового интерфейса.

frozzzen123 committed:
Сделал 1.2 1.3 1.4

Реализованы очки для пользователей, счетчик раундов и их вывод на экран.

23350406 committed:
Update pythonReady.cpp

Не забываем code style. Будет как в php

23350406 commented:
![2024-11-04_10-47-28](https://github.com/user-attachments/assets/948e50e2-2044-4a9d-bf46-3f4576cf1b0e)
